joplin/css/editor.scss

```diff
@@ -53,3 +53,9 @@ label {
         display: none;
     }
 }
+
```

Comment:
Where did this come from/whats it for?

Reply:
My IDE adds a newline at the end of every file I mess with.